### ISSUE TITLE
[BUG]: ReferenceError: getCapturesForExport is not defined

### ISSUE THREAD

**Issue by dwilches** [RevealBi/Reveal.Sdk#78]
### Company

N/A

### SDK Version

1.6.0

### Client Framework

Angular

### Server Platform

Java

### Description

Hello,
Today I noticed my server-side exported PDFs were failing, looking at the logs I found these exceptions. Not sure if they are related. This is an excerpt of the exceptions I see in the logs, the full exception is in the file attached.

```
2023-09-01 13:12:57,479 INFO  pool-4-thread-164 i.r.PlaywrightBrowser - Verifying playwright initialization: Is initialized = true {}
2023-09-01 13:12:57,800 INFO  pool-4-thread-164 i.r.PlaywrightBrowser - Evaluating function: () => getCapturesFor {}
2023-09-01 13:12:58,252 ERROR pool-4-thread-164 i.r.PlaywrightBrowser - Error evaluation function Error {
  message='ReferenceError: getCapturesForExport is not defined
    at eval (eval at evaluate (:191:30), <anonymous>:1:7)
    at UtilityScript.evaluate (<anonymous>:198:19)
    at UtilityScript.<anonymous> (<anonymous>:1:44)
  name='Error
  stack='Error: ReferenceError: getCapturesForExport is not defined
    at eval (eval at evaluate (:191:30), <anonymous>:1:7)
    at UtilityScript.evaluate (<anonymous>:198:19)
    at UtilityScript.<anonymous> (<anonymous>:1:44)
    at CRExecutionContext.evaluateWithArguments (/tmp/playwright-java-1986444697702411372/package/lib/server/chromium/crExecutionContext.js:103:33)
    at async Object.evaluateExpression (/tmp/playwright-java-1986444697702411372/package/lib/server/javascript.js:264:12)
    at async FrameManager.waitForSignalsCreatedBy (/tmp/playwright-java-1986444697702411372/package/lib/server/frames.js:151:20)
    at async FrameExecutionContext.evaluateExpressionAndWaitForSignals (/tmp/playwright-java-1986444697702411372/package/lib/server/dom.js:90:12)
    at async Frame.evaluateExpressionAndWaitForSignals (/tmp/playwright-java-1986444697702411372/package/lib/server/frames.js:816:19)
    at async FrameDispatcher.evaluateExpression (/tmp/playwright-java-1986444697702411372/package/lib/server/dispatchers/frameDispatcher.js:99:55)
    at async DispatcherConnection.dispatch (/tmp/playwright-java-1986444697702411372/package/lib/server/dispatchers/dispatcher.js:401:22)
 } {}
 com.microsoft.playwright.PlaywrightException: Error {
  message='ReferenceError: getCapturesForExport is not defined
    at eval (eval at evaluate (:191:30), <anonymous>:1:7)
    at UtilityScript.evaluate (<anonymous>:198:19)
    at UtilityScript.<anonymous> (<anonymous>:1:44)
  name='Error
  stack='Error: ReferenceError: getCapturesForExport is not defined
    at eval (eval at evaluate (:191:30), <anonymous>:1:7)
    at UtilityScript.evaluate (<anonymous>:198:19)
    at UtilityScript.<anonymous> (<anonymous>:1:44)
    at CRExecutionContext.evaluateWithArguments (/tmp/playwright-java-1986444697702411372/package/lib/server/chromium/crExecutionContext.js:103:33)
    at async Object.evaluateExpression (/tmp/playwright-java-1986444697702411372/package/lib/server/javascript.js:264:12)
    at async FrameManager.waitForSignalsCreatedBy (/tmp/playwright-java-1986444697702411372/package/lib/server/frames.js:151:20)
    at async FrameExecutionContext.evaluateExpressionAndWaitForSignals (/tmp/playwright-java-1986444697702411372/package/lib/server/dom.js:90:12)
    at async Frame.evaluateExpressionAndWaitForSignals (/tmp/playwright-java-1986444697702411372/package/lib/server/frames.js:816:19)
    at async FrameDispatcher.evaluateExpression (/tmp/playwright-java-1986444697702411372/package/lib/server/dispatchers/frameDispatcher.js:99:55)
    at async DispatcherConnection.dispatch (/tmp/playwright-java-1986444697702411372/package/lib/server/dispatchers/dispatcher.js:401:22)
 }
        at com.microsoft.playwright.impl.WaitableResult.get(WaitableResult.java:54)
        at com.microsoft.playwright.impl.ChannelOwner.runUntil(ChannelOwner.java:111)
        at com.microsoft.playwright.impl.Connection.sendMessage(Connection.java:113)
        at com.microsoft.playwright.impl.ChannelOwner.sendMessage(ChannelOwner.java:102)
        at com.microsoft.playwright.impl.FrameImpl.evaluateImpl(FrameImpl.java:308)
        at com.microsoft.playwright.impl.PageImpl.lambda$evaluate$22(PageImpl.java:666)
        at com.microsoft.playwright.impl.LoggingSupport.withLogging(LoggingSupport.java:47)
        at com.microsoft.playwright.impl.ChannelOwner.withLogging(ChannelOwner.java:87)
        at com.microsoft.playwright.impl.PageImpl.evaluate(PageImpl.java:666)
        at com.microsoft.playwright.Page.evaluate(Page.java:4066)
        at com.infragistics.reveal.engine.export.PlaywrightBrowser.evaluateFunctionInPage(PlaywrightBrowser.java:185)
        at com.infragistics.reveal.engine.export.ExportToolService.getImagesFromHtmlDashboard(ExportToolService.java:326)
        at com.infragistics.reveal.engine.export.ExportToolService.detailedExportToImage(ExportToolService.java:112)
        at com.infragistics.reveal.engine.export.DefaultDashboardExporter$3.onSuccess(DefaultDashboardExporter.java:293)
        at com.infragistics.reveal.engine.export.DefaultDashboardExporter.addProcessedWidget(DefaultDashboardExporter.java:562)
        at com.infragistics.reveal.engine.export.DefaultDashboardExporter.access$600(DefaultDashboardExporter.java:58)
        at com.infragistics.reveal.engine.export.DefaultDashboardExporter$5.handleResult(DefaultDashboardExporter.java:530)
        at com.infragistics.reveal.engine.export.DefaultDashboardExporter$5.handleResult(DefaultDashboardExporter.java:524)
        at com.infragistics.reveal.engine.RevealEngine$2.invoke(RevealEngine.java:164)
        at com.infragistics.reportplus.datalayer.VisualizationDataService.postProcessResult(VisualizationDataService.java:89)
        at com.infragistics.reportplus.datalayer.VisualizationDataService.access$500(VisualizationDataService.java:15)
        at com.infragistics.reportplus.datalayer.VisualizationDataService$1$1.invoke(VisualizationDataService.java:63)
        at com.infragistics.reportplus.datalayer.DataLayerService$3.invoke(DataLayerService.java:168)
        at com.infragistics.reportplus.datalayer.DataLayerService$6.invoke(DataLayerService.java:299)
        at com.infragistics.reportplus.datalayer.TabularDataService$3.invoke(TabularDataService.java:79)
        at com.infragistics.reportplus.datalayer.TabularDataService$6.invoke(TabularDataService.java:244)
        at com.infragistics.reportplus.datalayer.engine.adhoc.AdHocBuilder$1.invoke(AdHocBuilder.java:47)
        at com.infragistics.reportplus.datalayer.engine.adhoc.HierarchyDataBuilderBase$3$1.invoke(HierarchyDataBuilderBase.java:342)
        at com.infragistics.reportplus.datalayer.TabularDataService$8.invoke(TabularDataService.java:304)
        at com.infragistics.reportplus.datalayer.TabularDataService$24.invoke(TabularDataService.java:787)
        at com.infragistics.reportplus.datalayer.engine.db.NativeJdbcDbUtility.updateRecord(NativeJdbcDbUtility.java:224)
        at com.infragistics.reportplus.datalayer.engine.db.NativeJdbcDbUtility$4.invoke(NativeJdbcDbUtility.java:314)
        at com.infragistics.reportplus.datalayer.engine.db.NativeJdbcDbUtility$3.invoke(NativeJdbcDbUtility.java:231)
        at com.infragistics.reportplus.datalayer.engine.db.NativeJdbcDbUtility.getFieldValueFromSingleRecord(NativeJdbcDbUtility.java:102)
        at com.infragistics.reportplus.datalayer.engine.db.NativeJdbcDbUtility.existsRecord(NativeJdbcDbUtility.java:228)
        at com.infragistics.reportplus.datalayer.engine.db.NativeJdbcDbUtility.insertOrUpdateRecord(NativeJdbcDbUtility.java:310)
        at com.infragistics.reportplus.datalayer.engine.db.DbCacheStorage.storeData(DbCacheStorage.java:153)
        at com.infragistics.reportplus.datalayer.engine.TabularDataCacheService$3.invoke(TabularDataCacheService.java:118)
        at com.infragistics.reportplus.datalayer.engine.InMemoryCacheStorage.storeObject(InMemoryCacheStorage.java:77)
        at com.infragistics.reportplus.datalayer.engine.TabularDataCacheService.addData(TabularDataCacheService.java:108)
        at com.infragistics.reportplus.datalayer.TabularDataService.cacheData(TabularDataService.java:783)
        at com.infragistics.reportplus.datalayer.TabularDataService.cacheDataAndHandOff(TabularDataService.java:300)
        at com.infragistics.reportplus.datalayer.TabularDataService.access$300(TabularDataService.java:14)
        at com.infragistics.reportplus.datalayer.TabularDataService$5$2$2.invoke(TabularDataService.java:222)
        at com.infragistics.reportplus.datalayer.engine.InMemoryDataset.getTabularData(InMemoryDataset.java:155)
        at com.infragistics.reportplus.datalayer.TabularDataService$5$2.invoke(TabularDataService.java:217)
        at com.infragistics.reportplus.datalayer.TabularDataService$19.invoke(TabularDataService.java:641)
        at com.infragistics.reportplus.datalayer.engine.transformations.PivotTableTransformation$2$1.invoke(PivotTableTransformation.java:142)
        at com.infragistics.reportplus.datalayer.engine.pivot.PivotTableBuilder$1$1.invoke(PivotTableBuilder.java:292)
        at com.infragistics.reportplus.datalayer.engine.InMemoryDataset.copyToLoader(InMemoryDataset.java:216)
        at com.infragistics.reportplus.datalayer.engine.pivot.PivotTableBuilder$1.invoke(PivotTableBuilder.java:268)
        at com.infragistics.reportplus.datalayer.engine.TaskExecutionService$1.run(TaskExecutionService.java:47)
        at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
        at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
        at java.base/java.lang.Thread.run(Thread.java:834)
 Caused by: com.microsoft.playwright.impl.DriverException: Error {
  message='ReferenceError: getCapturesForExport is not defined
    at eval (eval at evaluate (:191:30), <anonymous>:1:7)
    at UtilityScript.evaluate (<anonymous>:198:19)
    at UtilityScript.<anonymous> (<anonymous>:1:44)
  name='Error
  stack='Error: ReferenceError: getCapturesForExport is not defined
    at eval (eval at evaluate (:191:30), <anonymous>:1:7)
    at UtilityScript.evaluate (<anonymous>:198:19)
    at UtilityScript.<anonymous> (<anonymous>:1:44)
    at CRExecutionContext.evaluateWithArguments (/tmp/playwright-java-1986444697702411372/package/lib/server/chromium/crExecutionContext.js:103:33)
    at async Object.evaluateExpression (/tmp/playwright-java-1986444697702411372/package/lib/server/javascript.js:264:12)
    at async FrameManager.waitForSignalsCreatedBy (/tmp/playwright-java-1986444697702411372/package/lib/server/frames.js:151:20)
    at async FrameExecutionContext.evaluateExpressionAndWaitForSignals (/tmp/playwright-java-1986444697702411372/package/lib/server/dom.js:90:12)
    at async Frame.evaluateExpressionAndWaitForSignals (/tmp/playwright-java-1986444697702411372/package/lib/server/frames.js:816:19)
    at async FrameDispatcher.evaluateExpression (/tmp/playwright-java-1986444697702411372/package/lib/server/dispatchers/frameDispatcher.js:99:55)
    at async DispatcherConnection.dispatch (/tmp/playwright-java-1986444697702411372/package/lib/server/dispatchers/dispatcher.js:401:22)
 }
        at com.microsoft.playwright.impl.Connection.dispatch(Connection.java:195)
        at com.microsoft.playwright.impl.Connection.processOneMessage(Connection.java:175)
        at com.microsoft.playwright.impl.ChannelOwner.runUntil(ChannelOwner.java:109)
        ... 55 more
2023-09-01 13:12:58,308 ERROR pool-4-thread-164 i.r.PlaywrightBrowser - Error taking screenshot + Error evaluation function Error {
  message='ReferenceError: getCapturesForExport is not defined
    at eval (eval at evaluate (:191:30), <anonymous>:1:7)
    at UtilityScript.evaluate (<anonymous>:198:19)
    at UtilityScript.<anonymous> (<anonymous>:1:44)
  name='Error
  stack='Error: ReferenceError: getCapturesForExport is not defined
    at eval (eval at evaluate (:191:30), <anonymous>:1:7)
    at UtilityScript.evaluate (<anonymous>:198:19)
    at UtilityScript.<anonymous> (<anonymous>:1:44)
    at CRExecutionContext.evaluateWithArguments (/tmp/playwright-java-1986444697702411372/package/lib/server/chromium/crExecutionContext.js:103:33)
    at async Object.evaluateExpression (/tmp/playwright-java-1986444697702411372/package/lib/server/javascript.js:264:12)
    at async FrameManager.waitForSignalsCreatedBy (/tmp/playwright-java-1986444697702411372/package/lib/server/frames.js:151:20)
    at async FrameExecutionContext.evaluateExpressionAndWaitForSignals (/tmp/playwright-java-1986444697702411372/package/lib/server/dom.js:90:12)
    at async Frame.evaluateExpressionAndWaitForSignals (/tmp/playwright-java-1986444697702411372/package/lib/server/frames.js:816:19)
    at async FrameDispatcher.evaluateExpression (/tmp/playwright-java-1986444697702411372/package/lib/server/dispatchers/frameDispatcher.js:99:55)
    at async DispatcherConnection.dispatch (/tmp/playwright-java-1986444697702411372/package/lib/server/dispatchers/dispatcher.js:401:22)
 } {}
 java.lang.Exception: Error evaluation function Error {
  message='ReferenceError: getCapturesForExport is not defined
    at eval (eval at evaluate (:191:30), <anonymous>:1:7)
    at UtilityScript.evaluate (<anonymous>:198:19)
    at UtilityScript.<anonymous> (<anonymous>:1:44)
  name='Error
  stack='Error: ReferenceError: getCapturesForExport is not defined
    at eval (eval at evaluate (:191:30), <anonymous>:1:7)
    at UtilityScript.evaluate (<anonymous>:198:19)
    at UtilityScript.<anonymous> (<anonymous>:1:44)
    at CRExecutionContext.evaluateWithArguments (/tmp/playwright-java-1986444697702411372/package/lib/server/chromium/crExecutionContext.js:103:33)
    at async Object.evaluateExpression (/tmp/playwright-java-1986444697702411372/package/lib/server/javascript.js:264:12)
    at async FrameManager.waitForSignalsCreatedBy (/tmp/playwright-java-1986444697702411372/package/lib/server/frames.js:151:20)
    at async FrameExecutionContext.evaluateExpressionAndWaitForSignals (/tmp/playwright-java-1986444697702411372/package/lib/server/dom.js:90:12)
    at async Frame.evaluateExpressionAndWaitForSignals (/tmp/playwright-java-1986444697702411372/package/lib/server/frames.js:816:19)
    at async FrameDispatcher.evaluateExpression (/tmp/playwright-java-1986444697702411372/package/lib/server/dispatchers/frameDispatcher.js:99:55)
    at async DispatcherConnection.dispatch (/tmp/playwright-java-1986444697702411372/package/lib/server/dispatchers/dispatcher.js:401:22)
 }
        at com.infragistics.reveal.engine.export.PlaywrightBrowser.evaluateFunctionInPage(PlaywrightBrowser.java:194)
        at com.infragistics.reveal.engine.export.ExportToolService.getImagesFromHtmlDashboard(ExportToolService.java:326)
        at com.infragistics.reveal.engine.export.ExportToolService.detailedExportToImage(ExportToolService.java:112)
        at com.infragistics.reveal.engine.export.DefaultDashboardExporter$3.onSuccess(DefaultDashboardExporter.java:293)
        at com.infragistics.reveal.engine.export.DefaultDashboardExporter.addProcessedWidget(DefaultDashboardExporter.java:562)
        at com.infragistics.reveal.engine.export.DefaultDashboardExporter.access$600(DefaultDashboardExporter.java:58)
        at com.infragistics.reveal.engine.export.DefaultDashboardExporter$5.handleResult(DefaultDashboardExporter.java:530)
        at com.infragistics.reveal.engine.export.DefaultDashboardExporter$5.handleResult(DefaultDashboardExporter.java:524)
        at com.infragistics.reveal.engine.RevealEngine$2.invoke(RevealEngine.java:164)
        at com.infragistics.reportplus.datalayer.VisualizationDataService.postProcessResult(VisualizationDataService.java:89)
        at com.infragistics.reportplus.datalayer.VisualizationDataService.access$500(VisualizationDataService.java:15)
        at com.infragistics.reportplus.datalayer.VisualizationDataService$1$1.invoke(VisualizationDataService.java:63)
        at com.infragistics.reportplus.datalayer.DataLayerService$3.invoke(DataLayerService.java:168)
        at com.infragistics.reportplus.datalayer.DataLayerService$6.invoke(DataLayerService.java:299)
        at com.infragistics.reportplus.datalayer.TabularDataService$3.invoke(TabularDataService.java:79)
        at com.infragistics.reportplus.datalayer.TabularDataService$6.invoke(TabularDataService.java:244)
        at com.infragistics.reportplus.datalayer.engine.adhoc.AdHocBuilder$1.invoke(AdHocBuilder.java:47)
        at com.infragistics.reportplus.datalayer.engine.adhoc.HierarchyDataBuilderBase$3$1.invoke(HierarchyDataBuilderBase.java:342)
        at com.infragistics.reportplus.datalayer.TabularDataService$8.invoke(TabularDataService.java:304)
        at com.infragistics.reportplus.datalayer.TabularDataService$24.invoke(TabularDataService.java:787)
        at com.infragistics.reportplus.datalayer.engine.db.NativeJdbcDbUtility.updateRecord(NativeJdbcDbUtility.java:224)
        at com.infragistics.reportplus.datalayer.engine.db.NativeJdbcDbUtility$4.invoke(NativeJdbcDbUtility.java:314)
        at com.infragistics.reportplus.datalayer.engine.db.NativeJdbcDbUtility$3.invoke(NativeJdbcDbUtility.java:231)
        at com.infragistics.reportplus.datalayer.engine.db.NativeJdbcDbUtility.getFieldValueFromSingleRecord(NativeJdbcDbUtility.java:102)
        at com.infragistics.reportplus.datalayer.engine.db.NativeJdbcDbUtility.existsRecord(NativeJdbcDbUtility.java:228)
        at com.infragistics.reportplus.datalayer.engine.db.NativeJdbcDbUtility.insertOrUpdateRecord(NativeJdbcDbUtility.java:310)
        at com.infragistics.reportplus.datalayer.engine.db.DbCacheStorage.storeData(DbCacheStorage.java:153)
        at com.infragistics.reportplus.datalayer.engine.TabularDataCacheService$3.invoke(TabularDataCacheService.java:118)
        at com.infragistics.reportplus.datalayer.engine.InMemoryCacheStorage.storeObject(InMemoryCacheStorage.java:77)
        at com.infragistics.reportplus.datalayer.engine.TabularDataCacheService.addData(TabularDataCacheService.java:108)
        at com.infragistics.reportplus.datalayer.TabularDataService.cacheData(TabularDataService.java:783)
        at com.infragistics.reportplus.datalayer.TabularDataService.cacheDataAndHandOff(TabularDataService.java:300)
        at com.infragistics.reportplus.datalayer.TabularDataService.access$300(TabularDataService.java:14)
        at com.infragistics.reportplus.datalayer.TabularDataService$5$2$2.invoke(TabularDataService.java:222)
        at com.infragistics.reportplus.datalayer.engine.InMemoryDataset.getTabularData(InMemoryDataset.java:155)
        at com.infragistics.reportplus.datalayer.TabularDataService$5$2.invoke(TabularDataService.java:217)
        at com.infragistics.reportplus.datalayer.TabularDataService$19.invoke(TabularDataService.java:641)
        at com.infragistics.reportplus.datalayer.engine.transformations.PivotTableTransformation$2$1.invoke(PivotTableTransformation.java:142)
        at com.infragistics.reportplus.datalayer.engine.pivot.PivotTableBuilder$1$1.invoke(PivotTableBuilder.java:292)
        at com.infragistics.reportplus.datalayer.engine.InMemoryDataset.copyToLoader(InMemoryDataset.java:216)
        at com.infragistics.reportplus.datalayer.engine.pivot.PivotTableBuilder$1.invoke(PivotTableBuilder.java:268)
        at com.infragistics.reportplus.datalayer.engine.TaskExecutionService$1.run(TaskExecutionService.java:47)
        at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
        at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
        at java.base/java.lang.Thread.run(Thread.java:834)
 Caused by: com.microsoft.playwright.PlaywrightException: Error {
  message='ReferenceError: getCapturesForExport is not defined
    at eval (eval at evaluate (:191:30), <anonymous>:1:7)
    at UtilityScript.evaluate (<anonymous>:198:19)
    at UtilityScript.<anonymous> (<anonymous>:1:44)
  name='Error
```

[reveal.log](https://github.com/RevealBi/Reveal.Sdk/files/12499040/reveal.log)

**Comment by brianlagunas** [comment]
I just tested on Windows and everything works on my end. What OS are you using?

**Comment by dwilches** [comment]
I'm using Ubuntu:
```
$ lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 20.04.6 LTS
Release:        20.04
Codename:       focal
```

This machine was working ok with version 1.4.0 earlier this week. I updated the Reveal version in the same machine, restarted my service, and the server-side export started failing.

**Comment by brianlagunas** [comment]
I just confirmed with the team. This was an issue we recently discovered and have already logged with AB#26506.

We'll get this fixed for you. Sorry for the inconvenience.

**Comment by dwilches** [comment]
Thank you!

**Comment by brianlagunas** [comment]
This has been fixed and will be available in our next release.

**Comment by dwilches** [comment]
@brianlagunas is it possible to have a 1.6.1 release sooner, before the 1.7.0 next month? We've been blocked by Issue #34 since May. Thanks

**Comment by brianlagunas** [comment]
Issue #34 has not been fixed as it is very complex issue for which we haven't found a good solution. Maybe you're confusing #34 with something else?

**Comment by dwilches** [comment]
Hello,
I'm referring to this fix of one of the bugs mentioned in #34:

> [AB#24804](https://infragistics.visualstudio.com/14a7928c-44bc-4aed-b2ca-9a0ffbb14d7a/_workitems/edit/24804) has been completed and fixed the bug with setting the series colors via the theme. It should be available in our Aug release.

So, setting our own colours with the method `PdfExportOptions.setTheme`, so they look like our graphs in the JS side.

Thanks

**Comment by brianlagunas** [comment]
That fix was released with our 1.6.0 release.

**Comment by brianlagunas** [comment]
v1.6.1 isn't scheduled for release until Oct 20. That schedule cannot be changed. However, if you'd be willing to use a nightly build, we can push a recent dev build for you to use to test the fix out. Keep in mind, this build will not go through the testing that a production build will get.

Are you okay with a nightly dev build?

**Comment by dwilches** [comment]
I can't release the nightly build to production, so we'll keep being blocked, but at least it will let me test the 2 fixes before October 20.
Thanks.

**Comment by brianlagunas** [comment]
We just pushed the `1.6.1-SNAPSHOT`. Give it a try and let us know if it fixes your problem.

![image](https://github.com/RevealBi/Reveal.Sdk/assets/835562/7a631596-47ef-4827-ab22-7bad5bc69c47)

**Comment by dwilches** [comment]
Hello,
I gave it a try, and this `ReferenceError: getCapturesForExport is not defined` is solved in version 1.6.1-SNAPSHOT.
Thanks.
